### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by ssteinbuss**
Adding the Apache 2.0 license file

### CODE REVIEW

**Comment by ssteinbuss** [comment]
Drafted some boilerplate text for this. @mohuber @anilturkmayali @Arian-IDSA @Sonia-IDSA any additions or modifications required?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2020 Fraunhofer-Institut für Software- und Systemtechnik
```
Comment (tmberthold) [inline LICENSE:178]: Is the following section intended or is it a copy-paste thing?

Comment (tmberthold) [reply LICENSE:178]: sorry my mistake, its the referenced appendix :)

Comment (ssteinbuss) [reply LICENSE]: fixed this one

**Review by mohuber — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+developer that takes on the issue. New issues should be initially marked with `status:open`.
+*  Basic labels: `bug`, `enhancement`, `suggestion`, `documentation` `outdated`, `question`, `discussion`
+*  `status:closed`: issue is closed (after successful approval by issuer and QA)
+*  `status:duplicate`: issue is a duplicate of another linked issue and therefore discontinued
+*  `status:in-progress`: issue has been assigned and is currently being worked on
+*  `status:on-hold`: issue may be implemented at a later date
+*  `status:open`: issue has been submitted or re-opened recently
+*  `status:out-of-scope`: issue is considered out of the project's scope and therefore not further considered
+*  `status:resolved`: issue has been implemented and tested by a developer
+*  `status:wont-fix`: issue is in scope but considered impossible or too expensive to deal with
```
Comment (tmberthold) [inline CONTRIBUTING.md:46]: The labels described here doesn't match the actual labels:
https://github.com/International-Data-Spaces-Association/IDS-testbed/labels

**Review by tmberthold — APPROVED** [review]

**Review by SebastianOpriel — APPROVED** [review]

**Review by SebastianOpriel — APPROVED** [review]

**Review by aitorce — APPROVED** [review]
Approved.

**Pull request merged**